tuna/example/tables.py

```diff
@@ -34,17 +36,17 @@
 class ExampleDBTables(DBTablesInterface):
   """Represents db tables for example lib"""
 
-  def __init__(self, **kwargs):
+  def __init__(self, **kwargs: Dict[str, Any]) -> None:
     """Constructor"""
     super().__init__(**kwargs)
 
     #for pylint
-    self.job_table = None
-    self.session_table = SessionExample
+    self.job_table: Job = None
+    self.session_table: SessionExample = SessionExample
```

Comment:
Would `SessionMixin` be a better typehint for session_table? @cderb

Reply:
That would be more general, but in this case it would always be the specialization, so this is fine.